site/content/en/docs/Integration Patterns/player-capacity.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Allocating based on GameServer Player Capacity"
+linkTitle: "Player Capacity"
+date: 2021-08-31
+weight: 90
+description: >
+  Find a `GameServer` that has room for a specific number of players.
+---
+
+{{< alpha title="Player Tracking, Allocation Player Filter and Allocation State Filter" gate="PlayerTracking,PlayerAllocationFilter,StateAllocationFilter" >}}
+
+Using this approach, we are able to be able to make a request that is akin to: "Find me a `GameServer` that is already
+allocated, with room for _n_ number of players, and if one is not available, allocate me a `Ready` `GameServer`".
+
+Common applications of this type of allocation are Lobby servers where players await matchmaking, or a 
+persistent world server where players connect and disconnect from a large map.
+
+<a href="../../../diagrams/allocation-player-capacity.puml.png" target="_blank">
+<img src="../../../diagrams/allocation-player-capacity.puml.png" alt="Player Capacity Allocation Diagram" />
+</a>
+
+## Example `GameServerAllocation`
+
+The below allocation will attempt to find an already Allocated `GameServer` from the `Fleet` "lobby" with room for 10 
+to 15 players, and if it cannot find one, will allocate a Ready one from the same `Fleet`. 
+
+```yaml
+apiVersion: "allocation.agones.dev/v1"
+kind: GameServerAllocation
+spec:
+  preferred:
+    - gameServerState: Allocated
+      matchLabels:
+        agones.dev/fleet: lobby
+      players:
+        minAvailable: 10
+        maxAvailable: 15
+  required:
+    matchLabels:
+      agones.dev/fleet: lobby
+```
+
+{{< alert title="Note" color="info">}}
+We recommend doing an extra check when players connect to a `GameServer` that there is the expected player capacity
+om the `GameServer` as there can be a small delay between a player connecting and it being reported
+to Agones.
+{{< /alert >}}
+
+## What's Next?
+
+- Have a look at all commands the [Client SDK]({{< ref "/docs/Guides/Client SDKs/_index.md" >}}) provides.
+* Check all the options available on [`GameServerAllocation`]({{% ref "/docs/Reference/gameserverallocation.md" %}}).
```

Comment:
I think that the rendering of the bulleted list has odd spacing because this list item uses a * instead of a -.

Reply:
Nice catch. That looks better now.